src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+export function uploadToS3(files) {
+  for (let i = 0; i < files.length; i++) {
+    let file = files[i];
+    let fileParts = files[i].name.split('.');
+    let fileName = fileParts[0];
+    let fileType = fileParts[1];
+    axios
+      .post('http://localhost:8082/api/S3Bucket/getSignedUrl', {
```

Comment:
create a new url const string in 
[`src/config/config.example.json`](https://github.com/SCE-Development/Core-v4/tree/dev/src/config/config.example.json) and use it like we do in [`src/APIFunctions/LedSign.js`](https://github.com/SCE-Development/Core-v4/blob/89a77d8ea0d8e5ab125fc26331fcdb4993352167/src/APIFunctions/LedSign.js#L54)

Reply:
She should just use the Mailer url constant right...not create a new one?